compiler/noirc_evaluator/src/ssa/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -326,13 +326,15 @@ impl AcirContext {
         // Compute the inverse with brillig code
         let inverse_code = brillig_directive::directive_invert();
 
-        let results = self.brillig(
+        let results = self.brillig_call(
             predicate,
-            inverse_code,
+            &inverse_code,
             vec![AcirValue::Var(var, AcirType::field())],
             vec![AcirType::field()],
             true,
             false,
+            0,
```

Comment:
Seems like we're saying that both inversion and quotient (all bit sizes) are going to be the unconstrained function with index zero?

Reply:
This is an unresolved index and is resolved only after acir generation is complete. I could add a comment to clarify this though.

Reply:
Would be good to have a `const PLACEHOLDER_BRILLIG_INDEX` to avoid needing the same comment in multiple places.

Reply:
Yeah I'll add that